packages/flutter/test/material/floating_action_button_location_test.dart

```diff
@@ -413,6 +579,53 @@ class _GeometryListenerState extends State<_GeometryListener> {
   }
 }
 
+const double _leftOffsetX = 44.0;
+const double _centerOffsetX = 400.0;
+const double _rightOffsetX = 756.0;
+const double _miniLeftOffsetX = _leftOffsetX - kMiniButtonOffsetAdjustment;
+const double _miniRightOffsetX = _rightOffsetX + kMiniButtonOffsetAdjustment;
+
+const double _topOffsetY = 56.0;
+const double _floatOffsetY = 500.0;
+const double _dockedOffsetY = 544.0;
+const double _miniFloatOffsetY = _floatOffsetY + kMiniButtonOffsetAdjustment;
+
+Widget _singleFABScaffold(
+  FloatingActionButtonLocation location,
```

Comment:
I believe the style for this is usually something like:

```
Widget _singleFABScaffold(
  FloatingActionButtonLocation location, {
  bool mini = false,
  TextDirection textDirection = TextDirection.ltr,
}) {
```

But it doesn't look like that's always the case, up to you

Reply:
I'd like to keep it like this for now. Keeping `{` and `}` on separate lines makes clear `git-diff`s when adding more required parameters (for instance, if another non-optional parameter is added after `location`).

I couldn't find any style guides for this situation.